packages/api-reference/src/types.ts

```diff
@@ -33,6 +33,12 @@ export type ReferenceConfiguration = {
   isEditable?: boolean
   /** Whether to show the sidebar */
   showSidebar?: boolean
+  /**
+   * Whether to show models in the sidebar, search, and content.
```

Comment:
10 points for jsdoc

Reply:
we should probably start adding defaults to all of our configs, then display the types somehow